ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -778,6 +778,17 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-nutanix-ipi-disconnected
+- as: nutanix-ipi-flow-networks-f60
```

Comment:
Now we already have 4.19 ci enabled, I think you need to rebase against upstream, so that add ci jobs for 4.19.